x11-terms/cool-retro-term/cool-retro-term-1.2.0.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils xdg
+
+QTW_PN="qmltermwidget"
+QTW_PV=63228027e1f97c24abb907550b22ee91836929c5
+QTW_P="${QTW_PN}-${QTW_PV}"
+
+DESCRIPTION="Terminal emulator with an old school look and feel"
+HOMEPAGE="https://github.com/Swordfish90/cool-retro-term/"
+
+SRC_URI="
+	https://github.com/Swordfish90/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/Swordfish90/${QTW_PN}/archive/${QTW_PV}.tar.gz -> ${QTW_P}.tar.gz
+"
+
+LICENSE="GPL-2 GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+RESTRICT="test"
```

Comment:
Why? Please add an upstream or gentoo bug to track status why tests are disabled. Or describe the reason if it can't be enabled in Gentoo.

Reply:
I am not sure what to do when the upstream does not have test: that is why I have added `RESTRICT="test"`.